go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	github.com/grpc-ecosystem/grpc-gateway/v2 v2.15.0
 	github.com/jackc/pgconn v1.13.0
 	github.com/jonboulle/clockwork v0.3.0
-	github.com/mattn/go-sqlite3 v2.0.3+incompatible
+	github.com/mattn/go-sqlite3 v1.14.16
```

Comment:
Is this technically a downgrade?

Reply:
Actually the latest version is 1.14.16 indeed. The v2 was released by mistaken. From their docs:

>Latest stable version is v1.14 or later, not v2. NOTE: The increase to v2 was an accident. There were no major changes or features.

This bump is necessary to fix compilation errors that came out from the sqlite bump.